### DISCUSSION TITLE
Chat about YOLOv5 pip maintenance

### DISCUSSION THREAD

**Discussion by glenn-jocher** [fcakyon/yolov5-pip#168] [Ideas]
@fcakyon hi there! I was hoping we could chat about streamlining YOLOv5 pip updates and maintenance. Could you pick a time here that works for you? Thanks!
https://calendly.com/ultralytics/meet

**Comment by fcakyon** [comment]
Hello @glenn-jocher, I cannot find any suitable slot in the given link. Available times are either too early or too late for me.

**Reply by glenn-jocher** [reply]
@fcakyon sorry, fixed to CET now! Try again :)
